src/frontend/components/DashboardVideoLiveControlPane/index.spec.tsx

```diff
@@ -17,12 +22,28 @@ jest.mock('data/appData', () => ({
 describe('<DashboardVideoLiveControlPane />', () => {
   // since there aren't any widgets for now, we expect DashboardVideoLiveControlPane to be empty
   it('renders DashboardVideoLiveControlPane', () => {
-    const { container } = render(
-      <ResponsiveContext.Provider value="large">
-        <DashboardVideoLiveControlPane video={mockVideo} />
-      </ResponsiveContext.Provider>,
+    const queryClient = new QueryClient();
+
+    render(
+      wrapInIntlProvider(
+        <QueryClientProvider client={queryClient}>
+          <ResponsiveContext.Provider value="large">
+            <DashboardVideoLiveControlPane video={mockVideo} />
+          </ResponsiveContext.Provider>
+        </QueryClientProvider>,
+      ),
     );
 
-    expect(container).toBeEmptyDOMElement();
+    // DashboardVideoLiveGeneralTitle
```

Comment:
You should add tests for the widgets you added (as what have been done for DashboardVideoLiveGeneralTitle)